app/src/main/java/com/example/concordia_campus_guide/LocationFragment/LocationFragment.java

```diff
@@ -112,8 +140,15 @@ public void onMapReady(GoogleMap googleMap) {
      * The purpose of this method is to figure the style of the map to display
      * @param googleMap is the map that is used in the application
      */
+    private void initFloorPlans() {
+        hallGroundOverlay = mMap.addGroundOverlay(mViewModel.getHallBuildingOverlay());
+        mbGroundOverlay = mMap.addGroundOverlay(mViewModel.getMBBuildingOverlay());
+        vlGroundOverlay = mMap.addGroundOverlay(mViewModel.getVLBuildingOverlay());
+    }
+
     private void setMapStyle(GoogleMap googleMap) {
         try {
+            googleMap.setIndoorEnabled(false);
```

Comment:
since we won't using the indoorMaps provided by google. Why don't we remove it from here ?

Reply:
from where? I am disabling it here...

Reply:
In UiSetting() method in locationFragment